gdal.yaml

```diff
@@ -1,10 +1,20 @@
 package:
   name: gdal
   version: 3.9.3
-  epoch: 0
+  epoch: 1
   description: GDAL is an open source MIT licensed translator library for raster and vector geospatial data formats.
   copyright:
     - license: MIT
+  dependencies:
+    provider-priority: 0
+
+data:
+  - name: py-versions
+    items:
+      3.10: "310"
+      3.11: "311"
+      3.12: "312"
+      3.13: "300"
```

Comment:
This looks odd - why 300?

Reply:
Do we need to add it then because will `300` need updating once 3.13 is ready?

Reply:
3.13 builds have been added for most of our existing Python packages. If someone were to use 3.13, then they'd be able to do so without having multiple versions installed. I'm indifferent, but this is how it's handled elsewhere. I think the idea is to mass update 3.13's priority to 313 from 300 when ready